examples/model_compress/pruning/mobilenet_end2end/prepare_data.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I saw this is mentioned in `.ipynb` text block, but did not in the code block, can we put code like `bash xxx.sh` in the code block? Because let users using two command tools is a little strange.

Reply:
fixed